examples/ENS/ENS.xml

```diff
@@ -40,7 +40,7 @@
   <ts:attribute-types>
     <ts:attribute-type id="name" syntax="1.3.6.1.4.1.1466.115.121.1.15">
       <ts:origins>
-        <ts:ethereum event="NameRegistered" filter="owner=${ownerAddress}" value="name"/>
+        <ts:ethereum event="NameRegistered" filter="label=${tokenID}" value="name"/>
```

Comment:
I thought the original is correct: `owner=${ownerAddress}`?

```
<ts:ethereum event="NameRegistered" filter="owner=${ownerAddress}" value="name"/>
```

But it doesn't look like the ENS name (or label, cost) is retrievable from the `NameRegistered` event which is defined as:

```
event NameRegistered(uint256 indexed id, address indexed owner, uint expires);
```

Reply:
@hboon there is more than one definition of that event, the other one is this: 

`    event NameRegistered(string name, bytes32 indexed label, address indexed owner, uint cost, uint expires);
`

tokenId is a decimal representation of the hexadecimal label hash and therefore can work

Reply:
@James-Sangalli Ah, I stand corrected. Thanks :)

Then:

`tokenId` instead of `tokenID`. The latter works in iOS, but only for compatibility.